_component_design/actions-menu.md

```diff
@@ -1,17 +1,110 @@
 ---
 title: Actions Menu
```

Comment:
General question across this whole doc: Is it `actions menu` or `action menu`? Please review and search-replace to use one or the other.

Reply:
I reread the documentation and decided that actions menu makes the most sense. I `:s/action menu/actions menu`